index.js

```diff
@@ -32,6 +36,10 @@ const processOptions = options => {
 	};
 };
 
+const getTargetDir = input => {
+	return path.resolve(typeof input === 'string' ? input : input.pathname);
```

Comment:
This is incorrect. The Node.js docs even point that out: https://nodejs.org/api/url.html#urlfileurltopathurl

Reply:
> This is incorrect. The Node.js docs even point that out: [nodejs.org/api/url.html#urlfileurltopathurl](https://nodejs.org/api/url.html#urlfileurltopathurl)

Thanks for letting me know. It seems I need more study Node modules.

Could I ask if we could use `fileURLToPath` here?
The document says it was added in 10.12.0, but this module's target node engines is greater than or equal to 10.
Is it acceptable to change this module's engine to 10.12.0?
Or do I need to add some polyfills to replace `fileURLToPath`?

Reply:
It's ok to use `fileURLToPath`. I plan to target Node.js 14 soon.